### ISSUE TITLE
🐛 BUG: Svelte scoped CSS is not working

### ISSUE THREAD

**Issue by alexbruno** [snowpackjs/astro#635]
### What package manager are you using?

NPM

### What operating system are you using?

WSL2 Ubuntu 20.04

### Describe the Bug

Svelte scoped CSS just not works

### Steps to Reproduce

1. `npm init astro` using any template
2. Create a Svelte component
3. Just put some scoped CSS at `<style>` block
4. Use the component at a page
5. Error! Scoped styles just not works.

**Comment by FredKSchott** [comment]
Sorry, we'll need more info in the steps to reproduce: I don't know enough about svelte to do this myself and know that we're talking about the same thing.

Can you copy an full example Svelte component file that isn't working in Astro?

**Comment by matthewp** [comment]
@alexbruno Just bumping so you know we need some more help on this one, thanks!

**Comment by alexbruno** [comment]
Sorry for the late response!

All CSS styles at a Svelte component `<style>` block are scoped by default.

I'm just trying to code a simple component like this:

```html
<script>
  export let name
  export let classes
</script>

<svg class="feather {classes}">
  <use href="/assets/feather.svg#feather-{name}" />
</svg>

<style lang="postcss">
  .feather {
    width: 1em;
    height: 1em;
    @apply inline-block stroke-0 stroke-current fill-current;
  }
</style>
```

Styles doesn't work.

I'm using PostCSS + Tailwind. But I made tests with vanilla CSS and it doesn't work too.

**Comment by vegetableman** [comment]
Reproducible to me as well. It works only after I kill the current watch and run `npm start` again.

**Comment by kelvinsjk** [comment]
I'm having a similar issue with scoped vanilla CSS in Svelte related to HMR and rehydration, though I don't need to kill the watch and restart the server: just a refresh in the browser will then update the styles. My problem only occurred for components requiring hydration; HMR worked fine for my static svelte components.

I fiddled around with the browser dev tools, and saw that the scoped styles live in two places: an external `Component.svelte.css` as well as a duplicate injected inline css (related to [issue #519](https://github.com/snowpackjs/astro/issues/519)?). Upon changing a file and HMR kicking in, the svelte-component got a new hash-identifier but the styles were still using the older hash-identifier. Refreshing in the browser updates the styles to match up.

**Comment by FredKSchott** [comment]
Thanks for the report! We believe that this has been fixed in astro v0.21, going out today. Let us know if you can still reproduce the issue after updating to v0.21 and we'll be happy to take a look.
